pages/apim/3.x/event-native-apim/event-native-apim-introduction.adoc

```diff
@@ -0,0 +1,129 @@
+[[event-native-apim-introduction]]
+= Introduction
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/event_native_apim_introduction.html
+:page-folder: apim/event-native-apim
+:page-layout: apim3x
+
+[label label-version]#New in version 3.20.0#
+[label label-version]#BETA release#
+
+== Event-driven asynchronous APIs
+
+Most developers are familiar with synchronous APIs, where the client sends a request to a server, and the server sends a response back. Every time the client needs new data, it has to send a request. You can think of synchronous APIs as a 'client pull' model. RESTful APIs are a type of synchronous API.
+
+In contrast, asynchronous APIs are event-driven and allow you to stream data. Asynchronous APIs can use callbacks, such as webhooks, if the requested data is not available at the time of the request. You can think of asynchronous APIs as a 'server push' model. AsyncAPIs are a type of asynchronous API.
+
+For more information about event-driven architectures and asynchronous APIs, see the link:https://www.asyncapi.com/[AsyncAPI website].
+
+== Event-native API management
+
+With the rise of cloud computing, many vendors started to build their solutions for the cloud. Rather than taking on-premises software and converting it to work as SaaS, it was built from the ground to to be cloud-based. These solutions are said to be cloud-native.
+
+Event-native API management is similar. Rather than taking a synchronous API management solution and converting it to handle event-driven APIs, an event-native API management solution is built from the ground up to handle event-driven APIs.
+
+== Gravitee's event-native API management
+
+Gravitee's API management solution can use an event-native policy execution mode, and can support all asynchronous, streaming, and event-driven API use cases. Of course, Gravitee's API management solution also supports synchronous use cases.
+
+Gravitee can act as a bridge between synchronous and asynchronous environments.
+
+For example, Gravitee can subscribe to a Kafka topic and then allow filtered HTTP GET subscriptions from clients as shown in the following diagram.
+
+image:{% link /images/apim/3.x/event-native/event-native-api-management-gateway.png %}[]
+
+== New policy execution mode (V4 BETA)
+
+Introduced in release 3.19.0 of the Gravitee API Management platform, there is a beta version of a new policy execution engine that enables an improved execution flow for synchronous APIs and supports event-driven policy execution for asynchronous APIs. It is based on a modern and fully reactive architecture designed to address a number of challenges Gravitee users have been facing with the existing policy execution engine, available with the Gravitee's V3 API definition specification used in versions prior to APIM 3.19.x.
```

Comment:
V4 engine is not only about policy execution, it changes many aspects of the execution.